Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build
+FROM node:16.14.0-slim AS build
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+
+COPY . .
+RUN npm run build
+
+# Run
+FROM node:16.14.0-slim
+WORKDIR /app
+
+ENV CONFIGS_PATH=/configs
+ENV NODE_ENV=production
+
+COPY package*.json ./
+RUN npm install --production
+COPY --from=build /app/dist ./dist
+
+VOLUME [ "/configs" ]
+
+USER node
+
+CMD ["node", "./dist/index.js"]
```

Comment:
More of a question, but @MarcusOtter do we emit source maps currently (.map.ts I think)? If so, maybe we should consider running with source maps enabled so if we get an exception, it'll give us the error from the source TS, and not the dist JS.

Reply:
I don't think we do, I always get errors pointing at the js files, have no clue about how to get source maps working

Reply:
@nchristopher Let's enable sourcemaps then:

`CMD ["node", "--enable-source-maps", "."`